Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[workspace]
+resolver = "2"
```

Comment:
what's the purpose of this line ?

Reply:
https://doc.rust-lang.org/edition-guide/rust-2021/default-cargo-resolver.html